DESCRIPTION

```diff
@@ -45,6 +45,7 @@ Imports:
     shinycssloaders (>= 1.0.0),
     shinyjs,
     shinyWidgets (>= 0.6.2),
+    teal.code (>= 0.4.1),
     teal.data (>= 0.3.0),
```

Comment:
This will have to be bumped, I guess.

Reply:
I wonder why I bumped teal.code instead of teal.data 🤔 I think I have mistaken